salt/etcd/etcd.conf.jinja

```diff
@@ -6,7 +6,7 @@
 ETCD_DATA_DIR="/var/lib/etcd/"
 ETCD_NAME="{{ grains['id'] }}"
 
-ETCD_LISTEN_CLIENT_URLS="https://0.0.0.0:2379"
+ETCD_LISTEN_CLIENT_URLS="https://127.0.0.1:2379"
```

Comment:
If you are changing this option please also change `ETCD_ADVERTISE_CLIENT_URLS` because etcd cluster will advertise that it is still listening on all interfaces which is not true.